icons/jsx/index.jsx

```diff
@@ -472,7 +472,6 @@ export { default as Check } from "./Check";
 export { default as CheckBoxIcon } from "./CheckBoxIcon";
 export { default as CheckBoxOutlineBlank } from "./CheckBoxOutlineBlank";
 export { default as CheckCircle } from "./CheckCircle";
-export { default as CheckCircleLightOutline } from "./CheckCircleLightOutline";
```

Comment:
I assume this in not intended? (same comment for `icons/es5/index.js` file)

I think someone manually added jsx and js files for this icon but did not add the actual svg file, so `npm run build` removes this this import...

Reply:
@viktorrusakov You are correct. Unfortunately, these lines will likely continue to be added until the appropriate SVG file is added...

Reply:
Yeah, I'm just worried about making a breaking change here in this PR. Can you maybe manually remove these changes, so we can safely merge this? And then we create a separate issue to add the missing SVG file.

Reply:
if we get https://github.com/openedx/paragon/pull/2869 merged we don't need to worry about this being a breaking change

Reply:
@brian-smith-tcril Yes, you're right, I merged it so there shouldn't be any problems now.

I took the liberty of updating this PR myself by rebasing it on `master` as I know @adamstankiewicz is quite busy with enterprise work and might not get the chance to get to this PR soon. I validated that running `npm run build` does not remove any icons anymore and that the fix works (this is one of the fixed icons that Adam added to the PR description)
![image](https://github.com/openedx/paragon/assets/52399399/4bdb9b42-f784-4adc-a465-6008510d748b)

I'm going to merge it in as it is quite an important fix and PR has been open for a long time now.